lib/cib/cib_ops.c

```diff
@@ -453,14 +453,16 @@ update_cib_object(xmlNode * parent, xmlNode * update)
 
     copy_in_properties(target, update);
 
-    crm_trace("Processing children of <%s id='%s'>",
-              pcmk__s(object_name, "<null>"), pcmk__s(object_id, "<null>"));
+    crm_trace("Processing children of <%s%s%s>", object_name,
+              ((object_id == NULL)? "" : " id="), pcmk__s(object_id, ""));
 
```

Comment:
at least it is consistent without the '' incl. below - yeah I can live with it ...